src/common.js.ts

```diff
@@ -252,89 +252,96 @@ function getErrorOrWarningFull(issue: OperationOutcomeIssue) {
+        if (issue.diagnostics.includes('http://loinc.org')) return false;
+        if (issue.diagnostics.includes('LOINC is not indexed!')) return false;
+        
+        //DICOM warnings can be ignored
+        if (issue.diagnostics.includes('ValueSet http://dicom.nema.org/')) return false;
+        
+        //Fragment codesystems can't be checked
+        if (issue.diagnostics.includes('Unknown code in fragment CodeSystem')) {
+            if (issue.diagnostics.includes('https://fhir.nhs.uk/CodeSystem/NHSDigital-SDS-JobRoleCode')) return false
         }
+        
+        
+        // these warnings need checking
         /*
-        if (issue.diagnostics.includes('Error HTTP 404')) {
-            // THis is issues with the Terminology Server not containig UKCore and NHSDigita CocdeSystems
-            if (issue.diagnostics.includes('https://fhir.nhs.uk/CodeSystem/Workflow-Code')) return false;
-            if (issue.diagnostics.includes('https://fhir.nhs.uk/CodeSystem/NHSDataModelAndDictionary-treatment-function')) return false;
+        if (issue.diagnostics.includes('Could not confirm that the codes provided are in the value set')) {
+            if (issue.diagnostics.includes('http://hl7.org/fhir/ValueSet/usage-context-type')) return false;
         }
-         */
         if (issue.diagnostics.includes("None of the codings provided are in the value set 'IdentifierType'")) {
             if (issue.diagnostics.includes('https://fhir.nhs.uk/CodeSystem/organisation-role')) return false;
         }
         if (issue.diagnostics.includes('The markdown contains content that appears to be an embedded HTML tag starting at')) return false;
 
-        // LOINC Related warnings
-        if (issue.diagnostics.includes('http://loinc.org')) return false;
-        if (issue.diagnostics.includes('The markdown contains content that appears to be an embedded HTML tag starting at')) return false;
-        if (issue.diagnostics.includes('LOINC is not indexed!')) return false;
         if (issue.diagnostics.includes('Error HTTP 403 Forbidden validating CodeableConcept')) return false;
-
-        if (issue.diagnostics.includes('Code system https://dmd.nhs.uk/ could not be resolved.')) return false
-/*
-        if (issue.diagnostics.includes('http://snomed.info/sct')) {
-            if (issue.diagnostics.includes('https://fhir.hl7.org.uk/ValueSet/UKCore-MedicationCode')) return false
-            if (issue.diagnostics.includes('https://fhir.hl7.org.uk/ValueSet/UKCore-VaccineCode')) return false
-        }
-*/
-        if (issue.diagnostics.includes('Unknown code')) {
-            if (issue.diagnostics.includes('https://fhir.nhs.uk/CodeSystem/NHSDigital-SDS-JobRoleCode')) return false
-        }
+        
         if (issue.diagnostics.includes('None of the codings provided are in the value set')) {
             if (issue.diagnostics.includes('https://fhir.nhs.uk/CodeSystem/NHSDigital-SDS-JobRoleCode')) return false
```

Comment:
There is no longer a codesystem named NHSDigital-SDS-JobRoleCode

Reply:
Agreed, But there IS a fixed value slice in the NHSDigital PractitionerRole profile, keeping this check in